src/win32-misc.h

```diff
@@ -24,6 +24,8 @@
 #ifndef __WIN32_MISC_H__
 #define __WIN32_MISC_H__
 
+#include <sdkddkver.h>
```

Comment:
I'm not familiar with this header, is it available on all windows platforms?

Reply:
it should probably be checked for existence in configure.ac

Reply:
> I'm not familiar with this header, is it available on all windows platforms?

Yes, It's a very common one.

Reply:
I'm pretty sure it's been available since Windows 2000. If we have any expectation of somebody trying to build it on Windows ME :laughing: then I guess it might need to be added to `configure.ac`